### PR TITLE
Add more memory range test cases

### PR DESCRIPTION
**Pull request by Dentosal**
Some of the cases around logd were not documented, see this spec PR: https://github.com/FuelLabs/fuel-specs/pull/468

### CODE REVIEW

File: fuel-vm/tests/log.rs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    let receipts = run_script(script.into_iter().collect());
+    assert_success(&receipts);
+
+    let mut receipts_it = receipts.into_iter();
+    for reg in 0..VM_REGISTER_COUNT {
+        if let Receipt::Log { ra, .. } = receipts_it.next().expect("Missing receipt") {
+            assert_eq!(ra, reg as u64);
+        } else {
+            unreachable!("Exptected a log receipt");
```
Comment (ControlCplusControlV) [inline fuel-vm/tests/log.rs:29]:
```suggestion
            unreachable!("Expected a log receipt");
```

**Review by ControlCplusControlV — COMMENTED** [review]
LGTM just some minor nits

**Review by freesig — APPROVED** [review]
Looks good, nice work

**Comment by Dentosal** [comment]
I've removed the jump tests here, those are covered in the relative jumps PR: https://github.com/FuelLabs/fuel-vm/pull/381

**Review by ControlCplusControlV — APPROVED** [review]

**Pull request merged**